CBP/client-assets/assets/configurations/feature/apps.json

```diff
@@ -721,22 +827,74 @@
       ],
       "name": "Contribution",
       "toolTip": "Contribution",
+      "hasRole": ["none"]
+    },
+
+    {
+      "featureIds": [
+        "feature_admin"
+      ],
+      "icon": "",
+      "iconType": "none",
+      "id": "group_admin",
+      "keywords": [
+        "admin"
+      ],
+      "name": "Admin",
+      "toolTip": "Admin",
       "hasRole": [
-        "content_creator",
-        "channel-creator",
-        "ka-creator",
-        "kb-creator",
+        "admin"
+      ]
+    },
+    {
+      "featureIds": [
+        "feature_mdo",
+        "feature_spv",
+        "feature_cbc",
+        "feature_igot"
+      ],
+      "icon": "",
+      "iconType": "none",
+      "id": "portal_admin",
+      "keywords": [
+        "admin"
+      ],
+      "name": "Portals",
+      "toolTip": "Portals",
+      "hasRole": [
+        "admin",
         "editor",
-        "content_reviewer",
         "content_publisher",
-        "admin"
+        "content_creator",
+        "content_reviewer",
+        "mdo_admin",
+        "cbp_admin",
+        "spv_admin",
```

Comment:
IN cbp roles, mdo_admin , sbp_admin & spv_admin shouldnt be there.

Reply:
Do review now